### PR TITLE
Treniformis

### LINKED ISSUE #26: Better name
`vessel-lists` is boring and non-descript.  Can we come up with something better?  Maybe more fishy?

### PR DESCRIPTION
**Pull request by bitsofbits**
In progress: DO NOT MERGE

closes: #26, #19
- [ ] Choose more unique name
- current working name is `treniformis` from _turbinaria reniformis_ or
  scroll coral. What comes on scrolls? Lists;-) 
- [ ] Make PIP installable
- Moved all files to be installed into `treniformis` directory
- Added setup.py and setup.cfg (for PiPy)
- Moved list that are primarily used to create other lists 
  into `treniformis/internal`.  Also moved out of date lists kept 
  around for historical reasons there.
- Created API so that lists not in `internal` are available
  by importing `treniformis`
- [ ] Document directory structure 
- [ ] Document tag release

### CODE REVIEW

**Comment by bitsofbits** [comment]
@geowurster, can you take a look at this when you get the chance. `pip install .` works and there is a UI to get at commonly used lists. I'm paricularly interested in feedback on:
- setup.py: does the way I'm pulling in data files look reasonable; plus anything else you see.
- directory structure (I'll work on documenting this more once we think it's reasonable)
- API. Currently lists are available in the top level namespace. If they are per year lists, there is dictionary for that list keyed on year, otherwise the list itself is present.  I subclassed dict/list so the help(OBJECT) includes the README file associated with the kind of list. For instance `help(treniformis.active_mmsis)` gives:

```
Help on active_mmsis in module treniformis object:

class active_mmsis(__builtin__.dict)
 |  # Active MMSIs 
 |  
 |  MMSIs of vessels which are broadcasting during the specified peroid
 |  
 |  Only MMSIs with a minimum number of positional reports are included
...
```
- tagging the release. Haven't thought about this; any thought you have would be appreciated.

**Comment by geowurster** [comment]
@bitsofbits You need to add `.ipynb_checkpoints/` to `.gitignore`.  GitHub's [default gitignore for Python](https://github.com/github/gitignore/blob/master/Python.gitignore) is pretty good.

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[metadata]
+description-file = README.md
```
Comment (geowurster) [inline setup.cfg]: @bitsofbits The readme needs to be translated to RST.  PyPi does not support Markdown.

File: treniformis/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# vessel-lists
```
Comment (geowurster) [inline treniformis/README.md:1]: @bitsofbits The readme needs to be translated to RST so it renders at PyPi, which does not support Markdown.

File: treniformis/__init__.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+* likely_fishing: vessels likely fishing based on AIS messages
+
+
+* known_likely_fishing: combination of known and likely MMSI lists
+
+	
+* spoofing: MMSI that appear to be associated with spoofing.
+
+
+# TODO
```
Comment (geowurster) [inline treniformis/__init__.py]: @bitsofbits Lets make this an actual issue rather than a todo in the docs.

**Comment by geowurster** [comment]
@bitsofbits Will finish reviewing tomorrow when I have more time.  Updating the gitignore will make this a lot easier.

**Comment by bitsofbits** [comment]
@kevin, thanks for looking things over; I updated `.gitignore`, translated `README` and moved the `TODO` to #30

**Comment by geowurster** [comment]
Superseded by [`treniformis`](https://github.com/globalfishingwatch/treniformis), specifically https://github.com/GlobalFishingWatch/treniformis/pull/1.

**Pull request closed without merging**